SSA/ElementTasks/src/task4FileParser/Main.java

```diff
@@ -0,0 +1,48 @@
+package task4FileParser;
+
+import utilities.IOUtilities;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+
+public class Main {
+    public static void main(String[] args) {
+        if (args != null && args.length > 1 && args.length < 4) {
+            try {
+                if (args.length == 2) {
+                    int count = countStringMatching(args[0], args[1]);
+                    IOUtilities.println(String.format(
+                            "Number of occurrences = %s", count));
+                } else {
+                    replaceInFile(args[0], args[1], args[2]);
+                }
+            } catch (IOException e) {
+                IOUtilities.println(String.format("Invalid file %s.",
+                        e.getMessage()));
+            }
+        } else {
```

Comment:
could we omit this `else ...` statement?

Reply:
fixed